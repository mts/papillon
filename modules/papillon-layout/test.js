import React from 'react';
import {
  container,
} from './flavor';

describe('<Label />', () => {
  let wrapper;

  describe('Instance', () => {
    test('must be an instance of Container', () => {
      wrapper = renderMount(container);
      expect(wrapper.find('Container')).toHaveLength(1);
    });
  });

  describe('Snaphot', () => {
    test('must match container', () => {
      expect(global.renderToJSON(container)).toMatchSnapshot();
    });
  });
});
