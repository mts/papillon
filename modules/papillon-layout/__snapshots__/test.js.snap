// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> Snaphot must match container 1`] = `
<div
  className="container border"
>
  Container
</div>
`;
